api/v1/db/oracledb/developers-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const { serializeDevelopers, serializeDeveloper } = require('../../serializers/developers-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+/**
+ * @summary Return a list of developers
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of developers
+ */
+const getDevelopers = async (queries) => {
+  const connection = await conn.getConnection();
+  const sqlParams = {};
+  if (queries.name) {
+    sqlParams.name = queries.name;
+  }
+  const sqlQuery = `SELECT ID, NAME, WEBSITE FROM DEVELOPERS 
+                   ${sqlParams.name ? 'WHERE NAME = :name' : ''}`;
```

Comment:
To format a multiline string, I'd suggest following this format: https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

Example: https://github.sig.oregonstate.edu/ecs-data/students-api-contrib/blob/master/contrib.js#L2-L8

Reply:
Changed to match standard format